### ISSUE TITLE
notificationd regularly crashes on sway

### ISSUE THREAD

**Issue by Richard-Rogalski** [lxqt/lxqt-notificationd#309]
##### Expected Behavior
No crashes

##### Current Behavior
When running the daemon for long enough, eventually it will segfault. Lots of notifications are sent to it on a normal day for me, not sure how to reproduce.

##### Context
Crashes under normal load regularly since I started using it

##### System Information
* Distribution & Version: Gentoo Unstable
* Kernel: 5.15.5-dist
* Qt Version: 5.15.2
* liblxqt Version: 1.0.0
* lxqt-build-tools Version: 0.10.0
* Package version: 1.0.0
* Running Sway as my compositor
* sway version: 1.7
* wlroots version: 0.15.1

Here's an example of a time it does it. I've reran the daemon through `time` this time for next time it segfaults.
<https://pastebin.com/jvfv8tF8>
If you want me to rebuild with debug opts let me know.

**Comment by tsujan** [comment]
We need a real backtrace. Consult your distro doc on how to find one.

**Comment by Richard-Rogalski** [comment]
```
QVariant::load: unable to load type 1032.
QVariant::save: unable to save type 'QDBusArgument' (type id: 1032).

QVariant::load: unable to load type 1032.

Thread 1 "lxqt-notificati" received signal SIGSEGV, Segmentation fault.
0x00007ffff7992ecc in ?? () from /usr/lib64/libQt5Widgets.so.5
```
This is the tail end of it, if you want the full one lmk (really just the first 2 lines repeating for infinity), hope this helps!

**Comment by tsujan** [comment]
It should be fixed in 1.2.0, after https://github.com/lxqt/lxqt-notificationd/commit/c3229fe445262db3a19dc93e04efd47b3ee851b3 — at least, it's fixed under Wayfire.

If you have 1.2.0 and still see the crash, please tell us to reopen this. A real backtrace would be welcome.

**Comment by tsujan** [comment]
Also, see https://github.com/lxqt/lxqt-notificationd/issues/329 (fixed by https://github.com/lxqt/lxqt-notificationd/commit/c3229fe445262db3a19dc93e04efd47b3ee851b3).
